package.json

```diff
@@ -93,11 +93,11 @@
     ]
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^5.3.21",
-    "@storybook/addon-knobs": "^5.3.19",
-    "@storybook/addon-notes": "^5.3.19",
+    "@storybook/addon-actions": "^6.2.9",
```

Comment:
These are major version changes. I don't see an explanation in the description as to why this is needed. Will you shed some light on this.